pythoncz/cli.py

```diff
@@ -0,0 +1,19 @@
+import os
+
+import click
+from flask.cli import FlaskGroup
+
+
+def create_app(info):
+    """
+    Development server wrapper so FLASK_APP and FLASK_DEBUG aren't necessary
+    by default, see http://flask.pocoo.org/docs/latest/cli/
+    """
+    from pythoncz.app import app
+    return app
+
+
+@click.group(cls=FlaskGroup, create_app=create_app)
+def main():
+    """This is a management script for the pythoncz application"""
+    os.environ['FLASK_DEBUG'] = os.getenv('FLASK_DEBUG', '1')
```

Comment:
Tahle řádka si zasluhuje komentář. Proč se to dělá?

Reply:
To je tam proto, aby měla `FLASK_DEBUG` env var jako výchozí hodnotu 1, protože si myslím, že to je přesně to, co od spouštění webu lokálně každý očekává. Ale chci dát možnost to overridnout, takže `FLASK_DEBUG=0 pythoncz run` by to vypnulo.

Třeba je na to dokonce nějaký trik, který hned každý pochopí i bez komentáře.

Reply:
Jinak ta řádka se nedá dát nikam později, musí se dát sem (learned the hard way).